app/views/content/train-to-be-a-teacher/what-is-qts.md

```diff
@@ -17,10 +17,25 @@ promo_content:
 navigation: 20.20
 navigation_title: Qualified teacher status (QTS)
 navigation_description: You need English qualified teacher status (QTS) to teach in maintained schools in England. Find out how to get QTS through teacher training.
-inset_text:
-  international-content:
-    text: If you qualified as a teacher outside the UK, you can teach in England for up to 4 years without QTS. However, having QTS may improve your chances of getting a teaching job in England.
-    color: grey
+expander:
+  qualified-outside-the-uk:
+    title: if you qualified as a teacher outside the UK
+    text: |-
+      <p>
+      If you qualified as a teacher outside the UK, you can teach in England for up to 4 years without QTS. However, having QTS may improve your chances of getting a teaching job in England.</p>
+      <p>
+      After 4 years, you'll need QTS to teach in many schools in England.</p>
+      <p>
+      <a href="/non-uk-teachers/teach-in-england-if-you-trained-overseas">Find out how to teach in England if you trained outside the UK</a>.
+      </p>    
+  get-qts-through-teacher-training:
+    title: get QTS through teacher training
+    text: |-
+      <p>
+      If live outside of the UK, you can either get <a href="/non-uk-teachers/teach-in-england-if-you-trained-overseas#english-qualified-teacher-status-qts">qualified teacher status (QTS)</a> or <a href="/non-uk-teachers/international-qualified-teacher-status">international qualified teacher status (iQTS)</a>.</p>
```

Comment:
```suggestion
      If you live outside the UK, you can either get <a href="/non-uk-teachers/teach-in-england-if-you-trained-overseas#english-qualified-teacher-status-qts">qualified teacher status (QTS)</a> or <a href="/non-uk-teachers/international-qualified-teacher-status">international qualified teacher status (iQTS)</a>.</p>
```

missing word

plus we usually say 'outside the UK' - but happy for you not to change this if you disagree!

Reply:
ah good spot. I've removed the QTS link as couldn't see any obvious non-uk content on QTS for trainees.